manifests/params.pp

```diff
@@ -17,21 +17,28 @@
   $version = 'latest'
   $service = 'jenkins'
   $install_from_git = false
+  $install_from_pip = true
+  $install_from_pkg = false
   $git_revision     = 'master'
   $git_url          = 'https://git.openstack.org/openstack-infra/jenkins-job-builder'
 
   case $::osfamily {
     'RedHat', 'Amazon': {
-      $python_packages = [ 'python', 'python-devel', 'python-pip', 'python-argparse']
+      $python_packages = [ 'python', 'python-devel', 'python-pip', 'python-argparse', 'PyYAML']
```

Comment:
As far as I'm aware, `python-argparse` doesn't exist on CentOS7 ...  see #12, which has a suggested fix for this.
Without the change in that PR, I get:
```
==> jenkins: Error: Execution of '/bin/yum -d 0 -e 0 -y list python-argparse' returned 1: Error: No matching Packages to list
==> jenkins: Error: /Stage[main]/Jenkins_job_builder::Install/Package[python-argparse]/ensure: change from absent to present failed: Execution of '/bin/yum -d 0 -e 0 -y list python-argparse' returned 1: Error: No matching Packages to list
```